frontend/src/lib-customizations/espoo/citizen/assets/i18n/fi.tsx

```diff
@@ -576,7 +576,7 @@ export default {
           info: {
             DAYCARE: [],
             PRESCHOOL: [
-              'Suomen ja ruotsin kielinen esiopetus alkaa 11.8.2022. Jos tarvitsette varhaiskasvatusta 1.8.2022 lähtien ennen esiopetuksen alkua, voitte hakea sitä tällä hakemuksella valitsemalla ”Haen myös esiopetukseen liittyvää varhaiskasvatusta”.'
+              'Suomen- ja ruotsin kielinen esiopetus alkaa 11.8.2022. Jos tarvitsette varhaiskasvatusta 1.8.2022 lähtien ennen esiopetuksen alkua, voitte hakea sitä tällä hakemuksella valitsemalla ”Haen myös esiopetukseen liittyvää varhaiskasvatusta”.'
```

Comment:
Shouldn't this be `Suomen- ja ruotsinkielinen`?

Reply:
Yes 🙈